src/plots/power_vega.jl

```diff
@@ -275,7 +289,7 @@ function plot_vega( case::Dict{String,<:Any};
 end
 
 
-function remove_information!(data::Dict{String,<:Any})
```

Comment:
This change is also a revert from the latest dev branch.

Reply:
Fixed (I think)